js/weekView.js

```diff
@@ -1,9 +1,159 @@
-"use strict"
+import { isSameDate, isSameWeek, getWeekDates, selectedDate } from "../js/dateManipulation.js"
 
-const calendarDays = document.querySelector('.weekView-main')
+const ROW_COUNT = 25
```

Comment:
I strongly suspect your row count is soooomehow related to hours. Like 24 hours in the day. And you will definitely need that constant later.
Also, the only place you use ROW_COUNT has - 1 on it. So maybe it's 24 afterall?